dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/BaseResource.java

```diff
@@ -67,6 +70,7 @@ public String version() {
     @Path("/metadata")
     @Public
     @ApiOperation(value = "Get FHIR Metadata", notes = "Returns the FHIR Capabilities statement for the application", response = CapabilityStatement.class)
+    @ApiResponses(@ApiResponse(code = 200, message = "successful operation", examples = @Example(@ExampleProperty(value = ""))))
```

Comment:
Tiny change to be consistent with other response messages:
```suggestion
    @ApiResponses(@ApiResponse(code = 200, message = "Successful operation", examples = @Example(@ExampleProperty(value = ""))))
```

Reply:
Good catch!